package.json

```diff
@@ -65,7 +65,7 @@
     "@octokit/rest": "^18.0.0",
     "@tsconfig/node10": "^1.0.7",
     "@types/jest": "^26.0.0",
-    "@types/node": "^10.17.35",
+    "@types/node": "^12.19.3",
```

Comment:
This is needed because `@types/node@^10` does not support globalThis, which is needed to use `global.browser`